src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATOR = 0;
+    private static final int AMOUNT = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = createReport(readFromFle(fromFileName));
+        writeToFile(toFileName, report);
+    }
+
+    public static String[] readFromFle(String fromFileName) {
```

Comment:
should this method and others be public and static?

Reply:
https://prnt.sc/114zfgy
They're not public and static local (haven't change it before screenshotting ofc)